src/commands/app/create.ts

```diff
@@ -89,7 +93,15 @@ export default class Create extends BaseCommand<typeof Create> {
     }
 
     try {
-      if (
+      if (this.flags.boilerplates) {
+        this.sharedConfig.boilerplateName = this.flags.boilerplates
+          .toLowerCase()
+          .replace(/ /g, "-");
+        this.sharedConfig.appName = await getAppName(
+          this.sharedConfig.boilerplateName
+        );
+        await this.boilerplateFlow();
```

Comment:
@harshithad0703 , please move this code in **flagsPromptQueue** function.

    before calling await this.flagsPromptQueue() add this line
        this.sharedConfig.boilerplate = this.flags.boilerplate;
        
    if (isEmpty(this.sharedConfig.boilerplate)) {
      this.sharedConfig.boilerplate = prompt to select boilerplate;
    }
     this.sharedConfig.boilerplateName = this.sharedConfig.boilerplate
          .toLowerCase()
          .replace(/ /g, "-");
        this.sharedConfig.appName = await getAppName(
          this.sharedConfig.boilerplateName
        );
    }
    
    We don't need this line as it's handled in flagsPromptQueue code https://github.com/contentstack/contentstack-apps-cli/blob/development/src/commands/app/create.ts#L100